spec/spec.md

```diff
@@ -79,50 +79,52 @@ The keywords MAY, MUST, MUST NOT, RECOMMENDED, SHOULD, and SHOULD NOT in this do
 ## Terminology
 
 [[def:Decentralized Identifier, Decentralized Identifier, DID, DIDs, DID Document, DID Documents]]
-~ A [W3C specification](https://www.w3.org/TR/did-core/) describing an _identifier that enables verifiable, decentralized digital identity_. Associated with a document containing properties outlined in the specification.
+~ A [W3C specification](https://www.w3.org/TR/did-core/) describing an _identifier that enables verifiable, 
+decentralized digital identity_. A DID identifier is associated with a JSON document containing cryptograhpic keys,
+services, and other properties outlined in the specification.
 
 [[def:DID Suffix, Suffix]]
-~ The unique identifier string within a DID URI. e.g. The unique suffix of `did:dht:123` would be `123`.
-
-[[def:DID Suffix Data, Suffix Data]]
-~ Data required to deterministically generate a DID, the [[ref:Identity Key]].
+~ The unique identifier string within a DID URI. For DID DHT the suffix is the [[ref:z-base-32]] encoded [[ref:Identity Key]].
 
 [[def:Identity Key]]
-~ An [[ref:Ed25519]] public key encoded with [[ref:z-base-32]] used to uniquely identify a `did:dht` document.
+~ An [Identity Key](#identity-key) is a [[ref:Ed25519]] public key encoded required to authenticate all records in 
```

Comment:
this reads weird. `is a [[ref:Ed25519]] public key encoded required` specifically. `are you missing `with [[ref:z-base-32]]`?

Reply:
yes good call out - thanks!